features/step_definitions/stage.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const { Before, Given, When, Then } = require('@cucumber/cucumber');
+const request = require('screwdriver-request');
+const sdapi = require('../support/sdapi');
+
+const TIMEOUT = 240 * 1000;
+
+Before(
+    {
+        tags: '@stage'
+    },
+    function hook() {
+        this.repoOrg = this.testOrg;
+        this.repoName = 'test-stage'; // functional-stage
+        this.buildId = null;
+        this.eventId = null;
+        this.pipelineId = null;
+        this.stageName = null;
+        this.stageId = null;
+        this.hubJobId = null;
```

Comment:
`this.hubJobId` may not be set.
We may need to add entry in below code block for this to get set
https://github.com/screwdriver-cd/screwdriver/blob/master/features/support/world.js#L135

Reply:
added